configuration-reference/table.md

```diff
@@ -245,7 +269,8 @@ Below is an example of setting AWS credential as part of table config using envi
       "onHeapDictionaryColumns": [],
       "varLengthDictionaryColumns": [],
       "segmentPartitionConfig": {
-        "pk": {
+        "columnPartitionMap": {
+          "column_foo": {
```

Comment:
Please fix the indentation and there is no closing brace

Reply:
I'll have to fix the indentation in a follow-up gitbook change once this is merged due to gitbook weirdness